web/views/icons/open-source.tmpl

```diff
@@ -0,0 +1,5 @@
+<svg fill="#000000" viewBox="-2 0 36 32" xmlns="http://www.w3.org/2000/svg" stroke="currentColor">
+	<path d="M-0.136 16.708c0.152-8.765 6.287-15.005 13.797-16.015 8.959-1.199 16.495 4.895 17.943 12.979 1.375 7.667-2.839 14.844-9.787 17.688-0.599 0.244-0.927 0.109-1.156-0.5l-3.453-8.969c-0.197-0.527-0.063-0.855 0.453-1.088 1.563-0.709 2.536-1.896 2.797-3.6 0.411-2.64-1.5-5.077-4.161-5.307-2.423-0.235-4.609 1.453-5 3.853-0.339 2.131 0.713 4.115 2.697 5.016 0.62 0.281 0.745 0.557 0.505 1.188l-3.469 9.031c-0.167 0.443-0.531 0.6-1 0.417-3.661-1.432-6.667-4.167-8.437-7.677-1.609-3.177-1.624-5.661-1.729-7.021zM1.213 16.584c0.027 0.427 0.041 0.921 0.084 1.427 0.405 4.64 3.197 9.26 8.452 11.817 0.209 0.093 0.287 0.052 0.365-0.145 0.959-2.527 1.927-5.052 2.901-7.579 0.083-0.208 0.041-0.307-0.152-0.427-2.041-1.287-3.057-3.131-2.943-5.552 0.063-1.391 0.6-2.615 1.537-3.636 1.932-2.109 4.968-2.568 7.453-1.135 2.052 1.187 3.197 3.484 2.916 5.839-0.235 1.968-1.244 3.479-2.953 4.5-0.172 0.104-0.224 0.187-0.145 0.389 0.979 2.532 1.953 5.063 2.916 7.595 0.079 0.203 0.157 0.244 0.36 0.145 2.297-1.068 4.208-2.599 5.688-4.64 2.244-3.115 3.171-6.579 2.728-10.391-0.88-7.584-7.703-13.865-16.489-12.781-6.844 0.839-12.604 6.615-12.719 14.573z"/>
+  /* Origin: https://www.svgrepo.com/svg/342086/open-source-initiative "GPL License"; i removed xml header and added color rules. I also changed viewBox since with stroke was quite broken (out of bounds) */
```

Comment:
Ditto.

Reply:
Given the official [Logo Usage Guidelines](https://opensource.org/logo-usage-guidelines/) this SVG must include the Trademark... and FWIW, they also have [an SVG file](https://opensource.org/osi-logo-files/) available that we can use.

Reply:
No comment. 🐸

![image](https://user-images.githubusercontent.com/18245694/236649342-d12665a0-f286-456c-b6ce-329150a42948.png)